### PR TITLE
Added Deformation Node

### PR DESCRIPTION
**Pull request by vicdoval**
Added new ~~Tension~~ Deformation node to measure variations between different vector states, it can be used to create a tension map or to create new deformations using the outputed info

tension
tension1
tension2

# Preflight checklist

Put an x letter in each brackets when you're done this item:

- [x] Code changes complete.
- [x] Code documentation complete.
- [x] Documentation for users complete (or not required, if user never sees these changes).
- [x] Manual testing done. 
- [ ] Unit-tests implemented.
- [x] Ready for merge.

### CODE REVIEW

**Comment by zeffii** [comment]
neat node/code :)

**Comment by vicdoval** [comment]
Im not sure if i should call this node "Tension" on "Deformation" ¿Any thoughts on this?

**Comment by zeffii** [comment]
to me, it seems `deformation` might be appropriate because the idea is to pass two states of the same mesh?
If one were to just provide a single object, and calculate the stresses based on joint interaction and gravity, then I think 'Tension' would be appropriate?

up to you.

File: nodes/analyzer/deformation.py [hunk 1]
```diff
@@ -107,85 +111,93 @@ def area_calc(np_verts, area_params):
...
         sinw('VerticesSocket', "Rest Verts")
         sinw('VerticesSocket', "Distort Verts")
         sinw('StringsSocket', "Edges")
         sinw('StringsSocket', "Pols")
 
-        sonw('StringsSocket', "Edges_Tension")
-        sonw('StringsSocket', "Pols_Tension")
-        sonw('StringsSocket', "Vert_Edge_T")
-        sonw('StringsSocket', "Vert_Pol_T")
+        sonw('StringsSocket', "Edges_Def")
```
Comment (zeffii) [inline nodes/analyzer/deformation.py]: underscores not necessary here

**Comment by enzyme69** [comment]
It's probably more like Tension node if it has attribute like Stretch etc. In the past I think there was an addon for Blender which kind of doing this and I use SV also to use it with.

**Pull request merged**